docs/src/modules/components/ReactRunner.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { useRunner } from 'react-runner';
+
+interface ReactRunnerProps {
+  code: string;
+  scope: {
+    process: {};
+    import: {};
+  };
+  onError: (error: string | null) => {};
+  forwardProps?: {};
+}
+
+// The docs https://github.com/nihgwu/react-runner
+export default function ReactRunner(props: ReactRunnerProps) {
+  const { code, scope: scopeProp, onError } = props;
+
+  let scope = scopeProp;
+
+  if (process.env.NODE_ENV !== 'production') {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    scope = React.useMemo(() => {
+      const handler = {
+        get() {
+          throw new Error(
+            [
+              'A demo tries to access process.x in ReactRunner. This is not supported.',
+              'If you do not need to show the source, you can set "hideToolbar": true to solve the issue.',
+            ].join('\n'),
+          );
+        },
+      };
+
+      const proxyProcess = new Proxy(scopeProp.process, handler);
+
+      return {
+        ...scopeProp,
+        process: proxyProcess,
```

Comment:
Why? I think `process` is supported

Reply:
In practice, the demos that use `process` can't be edited because he hides the source. The idea is to simplify the problem. Using `process` in the demos, might then fail once developers export to codesandbox or copy the source in their app.

Reply:
OK I get it, previously some react component in markdown is using `process` but it's not editable, so I added the support, I don't know if we still support such case, if so we can disallow it conditionally